tracer/src/Datadog.Trace.SourceGenerators/InstrumentationDefinitions/Sources.cs

```diff
@@ -17,24 +17,6 @@ internal static class Sources
 
         public static string CreateCallTargetDefinitions(IReadOnlyCollection<CallTargetDefinitionSource> definitions)
         {
-            static void BuildInstrumentationDefinitions(StringBuilder sb, List<CallTargetDefinitionSource> orderedDefinitions, string instrumentationsCollectionName)
```

Comment:
For a follow up PR, but I suspect we can _dramatically_ simplify the source generator definition now? 🤔

Reply:
Maybe. Now we only need to generate the `GetIntegrationIds `but for that it's very likely that we need to retrieve and generate all the `CallTargets`...